cpp/include/cudf/datetime.hpp

```diff
@@ -207,5 +207,89 @@ std::unique_ptr<cudf::column> is_leap_year(
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
 /** @} */  // end of group
+
+/**
+ * @brief  Round up to the nearest day
+ *
+ * @param[in] cudf::column_view of the input datetime values
```

Comment:
```suggestion
 * @param cudf::column_view of the input datetime values
```

Reply:
For all the previous functions in the file they all seemed to have`[in]`, should I still remove it?

Reply:
Just do it for your new code. The existing code will be cleanup later.